pages/finance.js

```diff
@@ -0,0 +1,82 @@
+import BarChartComponent from "../components/FinancialSummary/BarChartComponent"
+import ExpenseBreakdown from "../components/FinancialSummary/ExpenseBreakdown"
+import SponsorshipTiers from "../components/FinancialSummary/SponsorshipTiers"
+import SuccessStories from "../components/FinancialSummary/SuccessStories"
+import AsyncAPISummary from "../components/FinancialSummary/AsyncAPISummary"
+import OtherFormsOfFinancialSupport from "../components/FinancialSummary/OtherFormsOfFinancialSupport"
+import ContactUs from "../components/FinancialSummary/ContactUs"
+import NavBar from "../components/navigation/NavBar"
+import Head from "next/head"
+import StickyNavbar from "../components/navigation/StickyNavbar"
+import Container from "../components/layout/Container"
+import { useEffect, useState } from "react"
+
+function FinancialSummary() {
+  const [windowWidth, setWindowWidth] = useState(0);
+
+  const handleResize = () => {
+    setWindowWidth(window.innerWidth);
+  };
+  
+  // Update the window width when the component mounts and when the window is resized
+  useEffect(() => {
+
+    // Initial width
+    handleResize();
+
+    // Listen for window resize events
+    window.addEventListener('resize', handleResize);
+
+    // Clean up the event listener when the component unmounts
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const title = "AsyncAPI Finance Summary";
+  const description = "Financial Summary of AsyncAPI";
+
+  // Use Container only if the window width is more than 1700px
+  const shouldUseContainer = windowWidth > 1700;
```

Comment:
Why to use Container only for width > 1700px? `Container` component is compatible for mobile devices as well.

Reply:
Container element is added to address an unexpected issue  on wide screens. This addition helps maintain design consistency on wide screens but we dont have that issue on small screens